pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/impl/BacklogQuotaImpl.java

```diff
@@ -37,7 +38,7 @@ public static BacklogQuotaImplBuilder builder() {
     }
 
     public static class BacklogQuotaImplBuilder implements BacklogQuota.Builder {
-        private long limitSize;
+        private long limitSize = -1 * 1024 * 1024 * 1024;
```

Comment:
Why we need `-1 * 1024 * 1024 * 1024` not `-1`?

Reply:
the default backlog limit size is -1GB, we use `-1 * GB` and define `GB = 1024 * 1024 * 1024` is better.

Reply:
actually as long as it's negative, it doesn't really matter, but just want to keep it consistent with existing default value: https://github.com/apache/pulsar/blob/27dcb0a6febd0d88438e18fae72aad8ee38c0738/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BacklogQuotaManager.java#L57